fileio/ft_read_headshape.m

```diff
@@ -81,7 +83,7 @@
 coordsys       = ft_getopt(varargin, 'coordsys', 'head');    % for ctf or neuromag_mne coil positions, the alternative is dewar
 fileformat     = ft_getopt(varargin, 'format');
 unit           = ft_getopt(varargin, 'unit');
-
+image          = ft_getopt(varargin, 'image');               % path to .jpeg file
```

Comment:
could you add some default value for "image" when it is not specified by the user, e.g. if the input file is FILENAME.obj, that it checks whether FILENAME.jpg exists and if so that it is used as "image"?